packages/vue/src/components/atoms/SfArrow/SfArrow.vue

```diff
@@ -1,22 +1,36 @@
-<template>
-  <SfButton class="sf-arrow" type="button" v-bind="$attrs" v-on="$listeners">
+<template functional>
+  <component 
+    :is="injections.components.SfButton" 
+    :class="[data.class, data.staticClass, 'sf-arrow']" 
+    :style="[data.style, data.staticStyle]" 
+    type="button" 
+    v-bind="data.attrs" 
+    v-on="listeners"
+  >
     <!--@slot Use this slot to replace arrow icon-->
     <slot>
-      <SfIcon
+      <component 
+        :is="injections.components.SfIcon"
```

Comment:
similarly. I don't think it's required to use `component` for functional component.

Reply:
It's the same like above.